search-app/docs/api.md

```diff
@@ -3675,7 +3677,7 @@ Content-Length: 168
 ]
 ```
 
-On occassions when variable association cannot be processed at all due to invalid input, variable association request will return a failure status code with the appropriate error message.
+On occasions when variable association cannot be processed at all due to invalid input, variable association request will return a failure status code with the appropriate error message.
```

Comment:
ultra nit: `On occasions when variable association` -> `On occasions where a variable association`.
`variable association request` -> `the variable association request`.